bigbluebutton-html5/imports/ui/components/UserAvatar /UserAvatar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropsType from 'prop-types';
+
+
```

Comment:
remove this extra line, there should be only one empty line for any separation

Reply:
fixed